src/gluonts/evaluation/_base.py

```diff
@@ -262,11 +263,11 @@ def seasonal_error(
 
         seasonal_mae = np.mean(abs(y_t - y_tm))
 
-        return seasonal_mae if seasonal_mae is not np.ma.masked else np.nan
+        return nan_if_masked(seasonal_mae)
 
     def get_metrics_per_ts(
         self, time_series: Union[pd.Series, pd.DataFrame], forecast: Forecast
-    ) -> Dict[str, Union[float, str, None]]:
+    ) -> Dict[str, Any]:
```

Comment:
Yeah it turns out that was me being lazy: annotating the dictionary in the function body made this return annotation OK for mypy. But I'm not sure it was the `np.nan` addition that broke mypy, because `isinstance(np.nan, float) == True`. But it's fine, no change required here in the end.